packages/cli/bin/n8n

```diff
@@ -43,6 +43,9 @@ require('express-async-errors');
 require('source-map-support').install();
 require('reflect-metadata');
 
+// Loading dotenv early ensures that `process.env` is up-to-date everywhere in code
+require('dotenv').config();
```

Comment:
this is actually now loading env variables from `.env` files in e2e tests as well.